src/plugins/vis_augmenter/server/saved_objects/augment_vis.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { SavedObjectsType } from 'opensearch-dashboards/server';
+
+export const augmentVisSavedObjectType: SavedObjectsType = {
+  name: 'augment-vis',
+  hidden: false,
+  namespaceType: 'single',
+  management: {
+    // utilizing defaults for some of these fields
+    // icon: 'visualizeApp',
+    // defaultSearchField: 'id',
+    importableAndExportable: true,
+    getTitle(obj) {
+      return 'Augment-' + obj.attributes.visName;
```

Comment:
This will be the same for any augmentations linked to the same vis, right? That doesn't seem ideal. Do we have a plugin resource name we could also include for disambiguation? In either case, this isn't a blocker, but something we may want to open an issue and come back to later.

Reply:
yeah, I like that better!